CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+*       @nuclia/application
+*       @nuclia/lead
```

Comment:
Github compains about `@nuclia/lead`

Reply:
where is it complaining?

It seems right 

![Screenshot 2023-12-20 at 12 53 44](https://github.com/nuclia/frontend/assets/524413/0a103d7e-614d-40ec-a78a-0096a178f7db)

Reply:
when reviewing changed files, I saw this:
![Captura de Pantalla 2023-12-20 a les 18 57 47](https://github.com/nuclia/frontend/assets/52449656/b6200744-cce0-45e1-a76e-22603e25dd90)

Reply:
right
Nathan has just fixed it